### PR TITLE
Missing rewards soon tag

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Neurons that are about to start missing rewards need to be marked with a tag to draw users' attention, indicating that action needs to be taken to avoid missing rewards. This PR introduces a "warning" state to the `NeuronsTag` component and applies it to the tags for "missing soon" neurons.

# Changes

- Added a "warning" neuron tag status.
- Introduced a "...to confirm" tag.

# Tests

- Added.
- Tested manually.

image

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/lib/components/ui/NeuronTag.svelte [hunk 1]
```diff
@@ -5,8 +5,13 @@
   export let tag: NeuronTagData;
   export let size: "medium" | "large" = "medium";
 
-  let intent: "error" | "info";
-  $: intent = tag.status === "danger" ? "error" : "info";
+  let intent: "error" | "warning" | "info";
+  $: intent =
```
Comment (yhabib) [inline frontend/src/lib/components/ui/NeuronTag.svelte:9]: [nitpick] it is still manageable, but if you think we might have more states, I would refactor it into a simple if/else function.

File: frontend/src/tests/lib/utils/neuron.utils.spec.ts [hunk 2]
```diff
@@ -1665,6 +1669,15 @@ describe("neuron-utils", () => {
           ),
         },
       };
+      const losingRewardSoonNeuron: NeuronInfo = {
+        ...mockNeuron,
+        fullNeuron: {
+          ...mockNeuron.fullNeuron,
+          votingPowerRefreshedTimestampSeconds: BigInt(
+            nowInSeconds() - SECONDS_IN_HALF_YEAR + 10 * SECONDS_IN_DAY
```
Comment (yhabib) [inline frontend/src/tests/lib/utils/neuron.utils.spec.ts:1677]: [nitpick] wdyt of extracting this into a variable?

```
const timestampSixMonthsAgoPlus10Days = nowInSeconds() - SECONDS_IN_HALF_YEAR + 10 * SECONDS_IN_DAY
```

**Review by yhabib — APPROVED** [review]
Some minor things

File: frontend/src/tests/lib/utils/neuron.utils.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/utils/neuron.utils.spec.ts]: Good idea, added.

File: frontend/src/lib/components/ui/NeuronTag.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/ui/NeuronTag.svelte:9]: There are no plans yet, but if needed, it will definitely be improved.

**Pull request merged**